rifServices/src/main/java/org/sahsu/rif/services/datastorage/common/BaseSQLManager.java

```diff
@@ -168,6 +168,51 @@ public CachedRowSetImpl createCachedRowSet(
 		return cachedRowSet;			
 	}
 	
+	/** 
+	 * Create cached row set from AbstractSQLQueryFormatter.
+	 * No checks 0,1 or 1+ rows returned
+	 * 
+	 * @param connection,
+	 * @param queryFormatter,
+	 * @param queryName,
+	 * @param params
+	 *
+	 * @return CachedRowSetImpl cached row set
+	 */
+	@Override
+	public CachedRowSetImpl createCachedRowSet(
```

Comment:
It would almost certainly better to make this return the interface `CachedRowSet` here, rather than the implementation. Just in terms of general good practice.

Reply:
Done.